amount.go

```diff
@@ -116,25 +117,25 @@ func (a Amount) String() string {
 }
 
 // ToMinorUnits returns a in minor units.
-// Deprecated: Result may be undefined if the value can't be expressed as a 64-bit integer.
-// Will be removed in a future release.
+// If a cannot be represented in an int64, the result is undefined.
+//
+// Deprecated: Will be removed before v1.0.0. Use Int64() instead.
 func (a Amount) ToMinorUnits() int64 {
 	if a.number == nil {
 		return 0
 	}
 	return a.Round().number.Coeff.Int64()
 }
 
-// BigInt returns the integer value in minor units.
+// BigInt returns a in minor units, as a big.Int.
 func (a Amount) BigInt() *big.Int {
-	i := a.Round().number.Coeff
-	return &i
+	return &a.Round().number.Coeff
 }
 
-// Int64 returns the integer value of a in minor units.
-// Returns an error if value can't be expressed as a 64-bit integer.
+// Int64 returns a in minor units, as an int64.
+// If a cannot be represented in an int64, an error is returned.
 func (a Amount) Int64() (int64, error) {
-	n := *a.Round().number
+	n := a.Round().number
 	n.Exponent = 0
```

Comment:
This may change the underlying number.  I copied the value to avoid that.

Reply:
Whoops, that was unintentional.